src/photos.cs

```diff
@@ -1423,4 +1445,28 @@ interface PHCloudIdentifier : NSSecureCoding {
 		[Export ("initWithStringValue:")]
 		IntPtr Constructor (string stringValue);
 	}
+
+	[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof(NSObject))]
```

Comment:
```suggestion
	[BaseType (typeof (NSObject))]
```
small

Reply:
fixed! having seen both spacing styles all over (even in this file), I'm curious if this is the actual stylistic guideline we should stick with?